manifest.json

```diff
@@ -1,16 +1,15 @@
 {
-  "manifest_version": 2,
-
+  "manifest_version": 3,
   "name": "Toggle Grayscale",
   "description": "Toggles a grayscale filter on the active tab.",
   "short_name": "grayscale",
-  "version": "1.0",
+  "version": "2.0.0",
```

Comment:
major version bump due to changes in Chrome requirements (`scripting` requires Chrome 88 and Manifest v3+)